### ISSUE TITLE
Add support for Symfony 2.8+

### ISSUE THREAD

**Issue by willdurand** [nelmio/NelmioApiDocBundle#738]
This bundle does not support Symfony 2.8+ because of an error related to the DIC. We should fix it.
More info here: https://travis-ci.org/nelmio/NelmioApiDocBundle/jobs/87620636

**Comment by lyrixx** [comment]
I don't know how you are using the DIC; but it looks like you are trying to fetch a service that requires the kernel. So, you can write this kind of code:

```php
$kernel = new Kernel
$kernel->boot();
$container = $kernel->getContainer();
$container->set('kernel', $kernel);
$container->get('the service that depends on kernel');
```

**Comment by willdurand** [comment]
I dont know either :p Probably something related to the WebTestCase... Thanks for the hint :)

**Comment by willdurand** [comment]
@lyrixx just wondering, kernel is not injected in the DIC anymore?

**Comment by lyrixx** [comment]
I don't know ;)

**Comment by nicolas-grekas** [comment]
Setting a kernel is a workaround for a bug in Sf 2.8-beta1 that should be fixed by symfony/symfony#16609
